src/Components/Open/OpenModelDialog.jsx

```diff
@@ -1,20 +1,21 @@
-import React, {ReactElement, useState} from 'react'
-import Box from '@mui/material/Box'
+import React, {ReactElement} from 'react'
 import Button from '@mui/material/Button'
+import Box from '@mui/material/Box'
+import Divider from '@mui/material/Divider'
 import Stack from '@mui/material/Stack'
 import Typography from '@mui/material/Typography'
-import {useAuth0} from '../../Auth0/Auth0Proxy'
-import {checkOPFSAvailability} from '../../OPFS/utils'
-import {getFiles} from '../../net/github/Files'
+import GitHubFileBrowser from './GitHubFileBrowser'
```

Comment:
https://github.com/bldrs-ai/Share/wiki/Dev:-Style#imports

Reply:
so the imports for this package are the last?